lib/ae_mdw/db/sync/transaction.ex

```diff
@@ -277,6 +277,21 @@ defmodule AeMdw.Db.Sync.Transaction do
     ]
   end
 
+  defp tx_mutations(
+         :channel_create_tx,
+         _tx,
+         signed_tx,
```

Comment:
This param is only used by `:channel_create_tx`(and no other tx will use it). Could we move it outside of `tx_mutations` too? (following same reasoning as https://github.com/aeternity/ae_mdw/pull/426#discussion_r777557249)